keps/sig-node/1769-memory-manager/README.md

```diff
@@ -541,69 +542,49 @@ Yes, there is a number of Unit Tests designated for State file validation.
 _This section must be completed when targeting beta graduation to a release._
 
 * **How can a rollout fail? Can it impact already running workloads?**
-  Try to be as paranoid as possible - e.g., what if some components will restart
-   mid-rollout?
+  It is possible that the state file will have inconsistent data during the rollout, because of the kubelet restart, but
+  you can easily to fix it by removing memory manager state file and kubelet restart. It should not affect any running 
+  workloads.
+  
 
 * **What specific metrics should inform a rollback?**
+  None.
```

Comment:
Is it possible to have pods requesting numa alignment and the kubelet is unable to align them?  If so, how does a pod author know and how does an operator know?

Reply:
yes, it possible, in this case, the pod will fail and you will get the admission error under pod events. But I thought the question is mostly about the cluster rollback.

Reply:
> yes, it possible, in this case, the pod will fail and you will get the admission error under pod events. But I thought the question is mostly about the cluster rollback and I also unsure if the admission error can be referred to as a metric.

This is great information.  Even though it isn't a metric, please include it as an answer to this question.